### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by tribarstudio** [The-Poolz/Poolz-Back#97]
**Describe the bug**
The currency drop down menu images do not display on safari 

**To Reproduce**
1. Go to 'Currency tab'
2. Click on 'hover the menu to see icons not displaying for ETH / DAI'

**Expected behavior**
The logos should show 

**Screenshots**
![Screenshot 2021-01-07 at 10 37 41](https://user-images.githubusercontent.com/77101942/103883180-c8f94400-50d4-11eb-93ad-1901e7b08de6.png)

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser safari
 - Version 13.0.05

**Additional context**

**Wallet address**
0x88849a3Aa0d305f6Fa1EAC210477715FDA3e54A0

**Comment by tribarstudio** [comment]
![Screenshot 2021-01-07 at 10 45 13](https://user-images.githubusercontent.com/77101942/103883717-79ffde80-50d5-11eb-8bb9-e81d9f4d53a6.png)

also the same under tomochkain

**Comment by Lomet** [comment]
We need to check this out.
thanks for the report.

**Comment by evdmx** [comment]
> We need to check this out.
> thanks for the report.

This is because you are using 'WEBP' graphic files instead of svg or png graphics. Webp isn't supported by safari or some other browsers.

**Comment by tribarstudio** [comment]
Agreed I would use SVG for all your logo images to ensure they are resolution independent across all devices.
